.github/workflows/build.yml

```diff
@@ -109,10 +109,13 @@ jobs:
         with:
           name: lifecycle-windows-x86-64
           path: out/lifecycle-v*+windows.x86-64.tgz
+      - uses: azure/docker-login@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
I don't like seeing `WARNING! Your password will be stored unencrypted in /home/runner/.docker/config.json.` But wondering how we feel about introducing a new dependency - @jabrown85 WDYT?

Reply:
MS already owns github so I feel 🆗  on this new action.